third_party/txt/src/minikin/Layout.cpp

```diff
@@ -317,12 +315,13 @@ int Layout::findFace(const FakedFont& face, LayoutContext* ctx) {
   // Note: ctx == NULL means we're copying from the cache, no need to create
   // corresponding hb_font object.
   if (ctx != NULL) {
-    hb_font_t* font = getHbFontLocked(face.font);
+    bool isColorBitmapFont;
```

Comment:
Nit: I had been leaving comments indicating where the changes to minikin made for libtxt are in order to better track it in the future.

Reply:
Up to you if you wanted to continue that.